build.gradle

```diff
@@ -0,0 +1,55 @@
+plugins {
+    id 'java'
+    id 'checkstyle'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile 'com.itextpdf:itextpdf:5.5.9'
+    compile 'org.bouncycastle:bcpkix-jdk15on:1.63'
+    compile 'commons-io:commons-io:2.6'
+    compile 'org.apache.commons:commons-collections4:4.1'
+    compile 'javax.validation:validation-api:2.0.1.Final'
+
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.3.1'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.3.1'
+    testCompile 'org.mockito:mockito-core:3.0.0'
+    testCompile 'org.mockito:mockito-junit-jupiter:2.23.0'
+}
+
+test {
+    useJUnitPlatform()
+    maxHeapSize = '1G'
+}
+
+jar {
+  manifest {
+    attributes(
+      'Main-Class': 'com.advanon.pdfsignatures.Application'
+    )
+  }
+  from (configurations.compile.collect { entry -> zipTree(entry) }) {
+    exclude 'META-INF/MANIFEST.MF'
+    exclude 'META-INF/*.SF'
```

Comment:
itext lib has some signatures so that then "fat jar" does not work when itext is embedded into it. These lines do the trick and probably remove this signature from the build (but tbh I don't know for sure how does it work, just found it on the net)

This also should not affect license, since our lib is going to be open source as well :D